x/distribution/spec/01_concepts.md

```diff
@@ -3,12 +3,21 @@ order: 1
 -->
 
 # Concepts
+In Proof of Stake (PoS) blockchains, rewards gained from transaction fees are paid to validators. The fee distribution module fairly distributes the rewards to the validators' constituent delegators. 
+
+Rewards are calculated per period. The period is updated each time a validator's delegation changes, for example, when the validator receives a new delegation.
+The rewards for a single validator can then be calculated by taking the total rewards for the period before the delegation started, minus the current total rewards.
+To learn more, see the [F1 Fee Distribution paper](/docs/spec/fee_distribution/f1_fee_distr.pdf).
+
+The commission to the validator is paid when the validator is removed or when the validator requests a withdrawal.
```

Comment:
Does this mean that commissions do not compound a validator's reward?

Reply:
Yes. The commission is just stored in the distribution module account, it is not put towards any delegation automatically anywhere I can see. Is there a hook I'm overlooking?